### ISSUE TITLE
Training NAN

### ISSUE THREAD

**Issue by whcjb** [sfzhang15/RefineDet#58]
Hi, Zhang
I use the RefineDet architecture to train on widerface, the pretrained model is vgg.
when the iteration round 40, it comes out NAN.
the first iteration loss is about 10plus.
can't figure out the reason, could you give a propsal?
thank you!

**Comment by whcjb** [comment]
it seems to be the GPU problem, when four gpus used, it comes normally. The batch size limit to 16, however.
Thanks.

**Comment by whcjb** [comment]
oh, I think it is time to change to Titan X now.
Thinks.

**Comment by kirnh** [comment]
@whcjb , I am facing the same issue. The loss becomes NaN after 30 iterations. Are you saying its a batchsize issue? @sfzhang15 , can you please give some clarity on this?

**Comment by sfzhang15** [comment]
@kirnh 
If training some iterations and getting NaN, this is because the pace of learning is too big. You can try to reduce learning rate, or use small scale parameter for the normlization layer.
